lib_comfyui/parallel_utils.py

```diff
@@ -50,17 +50,23 @@ def __setstate__(self, state):
         *state, self._consumer_ready_event, self._producer = state
         return super(SynchronizingQueue, self).__setstate__(state)
 
-    def start_thread(self):
+
+class AsyncProducerHandler:
+    def __init__(self, queue: SynchronizingQueue):
+        self.queue = queue
+        self.producer_thread = None
+
+    def start_producer_thread_loop(self):
```

Comment:
why not just `start` and `stop`? These methods are starting to look a lot like the interface of a thread. I'm usually not a fan of deep inheritance trees but in this case it could make sense to go for that... idk

Reply:
I will rename to `start` and `stop`